### DISCUSSION TITLE
Multiple Network Cards on LTSP Server

### DISCUSSION THREAD

**Discussion by JB-Prof** [ltsp/ltsp#226] [Q&A]
Hello,

I hope you are doing very well, and a happy New Year to you all!

I am in the process of installing LTSP 19.12.  I'm using the base OS of Desktop Ubuntu-Mate, as recommended.

Our network setup is such that we have all the thin clients on their own subnet with their own switches.  So we need the two-NIC setup - one for LTSP, the other for Internet into our router.

I have 4 network ports on this server and I'm thinking about the best way to set it up for performance.  We have about 80 LTSP clients booting off this server all day.

Is it possible to aggregate two or three NICs (also known as NIC bonding I think) for LTSP traffic so I get better performance?  The switch the server connects to also does NIC aggregation so it can handle whatever we do on the server side.

If yes, this is a good idea, are there specific instructions for doing this with LTSP?

Thank you so much.

**Comment by alkisg** [comment]
Happy new year to all,

an easy and fast solution would be to use the [Network Manager GUI](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/networking_guide/sec-creating_a_bond_connection_using_a_gui) to set up a bond between 3 of the adapters, and assign a static IP of 192.168.67.1 to the bond.

After that, do a LAN benchmark via Epoptes; you should see more than 2 Gbps of upload/download rate.

Remember to put the following in ltsp.conf, for the client to have Internet access:

```shell
[server]
NAT=1
```

**Comment by JB-Prof** [comment]
Hello,

Thank you for the response.  

Would I create this bond before or after the LTSP-specific steps indicated in the installation setup?  i.e.:  I currently have the base OS set up and all configured, and I am ready to turn it into an LTSP server - do I do this bonding now or after it is an LTSP server?

Thank you.

**Comment by alkisg** [comment]
It doesn't matter, you can do it either before of afterwards.

After you put NAT=1 in ltsp.conf, you either need to reboot the server, or run `systemctl restart ltsp`, for the IP forwarding to take effect.
